site/content/integrate/apps/bindings/_index.md

```diff
@@ -5,21 +5,23 @@ description: "TODO"
 weight: 50
 ---
 
-Bindings ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Binding)) are what establish the relationship between locations and calls. Whenever it's called, you have to provide the list of bindings available according to the context. The context for the bindings call includes the app ID, bot access token, team ID, channel ID, acting user ID and Mattermost site URL. By default it doesn't expand any value.
+Bindings ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Binding)) are what establish the relationship between locations and calls. Whenever it's called, you have to provide the list of bindings available according to the context. The context for the bindings call includes the app ID, bot access token, team ID, channel ID, acting user ID, and Mattermost site URL. By default it doesn't expand any value.
```

Comment:
```suggestion
Bindings ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Binding)) are what establish the relationship between locations and calls. Whenever the bindings route is called, your app needs to provide the list of bindings available according to the context ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Context)). Some fields included in the context:
- Your app's bot user access token
- The Mattermost Site URL
- The id of the user requesting the bindings (acting user id)
- The id of the team the user is currently focused on
- The id of the channel the user is currently focused on
- The id of the post the user is currently focused on (if applicable)
```

> By default it doesn't expand any value.

I'm not sure if we explain `expand` anywhere. Saying "By default it doesn't expand any value." doesn't mean anything if `expand` isn't itself explained